Touch-up Python code and tweak some grammar

brettcannon opened the pull request:
Also clarified that pip may not be required to be separately installed and made the installation of `cookiecutter` not a global one.

azurecla commented:
Hi __@Saisang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

brettcannon committed:
Touch-up Python code and tweak some grammar

Also clarified that pip may not be required to be separately installed and made the installation of `cookiecutter` not a global one.
